src/components/ratings.js

```diff
@@ -1,44 +1,48 @@
 import React, { Component } from 'react';
-import {connect} from 'react-redux';
-
+import { connect } from 'react-redux';
 import Ratings from 'react-ratings-declarative';
 
 import { updateBeer } from '../actions/index';
 
 export var ratingChange = false;
 
 class RatingsComponent extends Component {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            rating: props.beerRating
-        }
-
-        this.changeRating = this.changeRating.bind(this);
-    }
-
-    changeRating(id, rating){
-        this.props.updateBeer(id, rating);
- ratingChange = true;
-        this.setState({
-            rating: rating
-        })
-    }
-
-    render() {
-        return (
-            <Ratings rating={this.state.rating}
-                widgetRatedColors="blue"
-                changeRating={(rating) => { this.changeRating(this.props.beerId, rating) }}>
-                <Ratings.Widget />
-                <Ratings.Widget />
-                <Ratings.Widget />
-                <Ratings.Widget />
-                <Ratings.Widget />
-            </Ratings>
-        )
-    }
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      rating: props.beerRating,
+    };
```

Comment:
https://babeljs.io/docs/en/babel-plugin-proposal-class-properties

You can use this, its supported

Reply:
I've implemented it based on what I understood from the docs, but I'm not sure if it's correct.

Reply:
You dont need `call()`. Just `this.getRating()`